### DISCUSSION TITLE
How can I add or install an windows game in a Wine Prefix?

### DISCUSSION THREAD

**Discussion by ghost** [tkashkin/GameHub#460] [Q&A]
So I wanna add and install some windows games,and I wanna add/install them to a prefix.
So how can I do that?
I get that I have to set the game/installer executable [here](https://i.imgur.com/g2bzECZ.png) at "Executable",but what does "Directory" mean there?
If I set an Directory there,will it create an Wine Prefix there?
And if so,if it's an installer,will it automatically open the installer and I can install it in the prefix or how does it work?
And if I wanna add an game,will it then automatically be copied with all the files to the prefix?
And how will that change,if I want to add an Linux executable?
Will it then just create a normal folder and copy the files to that directory?

**Comment by neuromancer** [comment]
> If I set an Directory there,will it create an Wine Prefix there?

Yes.

>  And if so,if it's an installer,will it automatically open the installer and I can install it in the prefix or how does it work?

Yes.

> And if I wanna add an game,will it then automatically be copied with all the files to the prefix?

How you will add it exactly? Using installer is the correct way to way to do it.

**Reply by ghost** [reply]
> How you will add it exactly? Using installer is the correct way to way to do it.

What I mean is,if I want to add an exectutable that isn't an installer.
I mean if the executable that I wanna add launches the game itself.
So is there a way to copy that executable with all the files that are needed to work to the selected wine prefix? (so it copies all the files in the games folder to a folder in the prefix [like "C:>Program Files>(name of the game)"]) or something like that,is that possible?

**Comment by tkashkin** [comment]
> I get that I have to set the game/installer executable here at "Executable",but what does "Directory" mean there?

Directory here means the directory where game is already installed or directory where game is expected to be installed by the Installer.

> If I set an Directory there,will it create an Wine Prefix there?

Wine/Proton prefix will be created in `_gamehub` subdirectory of game's installation directory. (That might be changed in the future release)
You can override prefix location in game's compatibility options.

> And if I wanna add an game,will it then automatically be copied with all the files to the prefix?

It will not be copied. Prefix is created inside of selected directory by default and GameHub tries to mount the installation directory as `D:\` drive in Wine to simplify the installation process.

> And how will that change,if I want to add an Linux executable?

Game files are not copied anywhere in both cases, so nothing changes.
